### PR TITLE
Source code viewer build process and webpack optimizations

### PR DESCRIPTION
**Pull request by DanWahlin**
This PR does the following:

1. Creates a source viewer build process as a standalone script. Generates rawSources.json file and puts it in /public.
2. Updates package.json build scripts to call this new source viewer build script.
3. Fixes a minor bug with the drop-down in the source viewer.
4. Cleans up older webpack loader code that originally generated the rawSources.json file.
5. Modifies the Fluid Playground link that displays in the upper-left corner
6. Updates Fluid packages to 0.26.0
7. Added new source viewer build script into webpack so it 'll run anytime webpacks rebuilds /src

Does the CI/CD process still work with the latest move from SWA to blob storage? Wasn't sure if the same blob storage was being used for both or if things changed there.

## Verification Steps

Clone this branch and run:

1. `npm start` (app should load fine and you should see a rawSources.json in /public that's been updated).
2. `npm run build:hugo` (storybook-static folder that's generated should have a rawSources.json file). I normally run `npx http-server` from within the `storybook-static` folder to try it.

Thanks to Captain Jokiel (@skylerjokiel) for the initial look at optimizations, the main.js webpack contribution, and for triggering the idea for this new build/optimization approach.

### CODE REVIEW

File: .storybook/main.js [hunk 1]
```diff
@@ -3,53 +3,18 @@
...
-    // Used by sourcecode-addon. Add loader that registers raw source code in a cache and display it in the Source tab.'
-    config.module.rules.push({
-      test: /\.jsx$|\.tsx$|\.css$|\.ts$|\.js$|\.mdx$/,
-      use: [
-        {
-          loader: path.resolve(__dirname, 'sourcecode-addon/webpackLoader.js'),
-          options: { root: path.resolve(__dirname, '../src') },
-        },
-      ],
-    });
```
Comment (skylerjokiel) [inline .storybook/main.js:26]: Don't you still need these additional rules for building the stories?

**Review by skylerjokiel — APPROVED** [review]
Amazing Dan! 

I left one comment about pulling out the additional webpack build rules.

File: .storybook/main.js (hunk 1 above)
Comment (DanWahlin) [reply .storybook/main.js:26]: No - those are just for building up the rawSources.json file. Since that's move into a separate script now it can go.

**Review by SamBroner — APPROVED** [review]
Big bundle size improvement! Nice Dan.

I'm not. familiar with that hook, but I like that both npm run start and npm run build work well.

**Comment by SamBroner** [comment]
BTW, to my tests: 

Before:
51.6 MB all up in static folder (all assets, not a great metric
main.a95189b9a8d29ba93d03.bundle.js (590 KiB)
vendors~main.a95189b9a8d29ba93d03.bundle.js (16.1 MiB)
rawSources.json (245 KiB)
main (16.7 MiB)

After:
24.7 MB all up in static folder
main.505ab70d388cb9a8d7b7.bundle.js (245 KiB)
vendors~main.505ab70d388cb9a8d7b7.bundle.js (5.82 MiB)
main (6.09 MiB)

**Comment by DanWahlin** [comment]
Thanks for your help guys!

**Pull request merged**